toolchain/check/handle_class.cpp

```diff
@@ -346,13 +346,13 @@ static auto DiagnoseClassSpecificDeclRepeated(Context& context,
                                               SemIRLoc prev_loc,
                                               Lex::TokenKind tok) -> void {
   CARBON_DIAGNOSTIC(ClassSpecificDeclRepeated, Error,
-                    "Multiple `{0}` declarations in class.{1}", Lex::TokenKind,
+                    "multiple `{0}` declarations in class.{1}", Lex::TokenKind,
                     std::string);
   const llvm::StringRef extra = tok == Lex::TokenKind::Base
                                     ? " Multiple inheritance is not permitted."
```

Comment:
```suggestion
                    "multiple `{0}` declarations in class{1}", Lex::TokenKind,
                    std::string);
  const llvm::StringRef extra = tok == Lex::TokenKind::Base
                                    ? "; multiple inheritance is not permitted"
```
(or revert the changes to this file and merge #4327).